configure.py

```diff
@@ -633,9 +647,11 @@ def has_re2c():
     print('bootstrap complete.  rebuilding...')
     if platform.is_windows():
         bootstrap_exe = 'ninja.bootstrap.exe'
-        if os.path.exists(bootstrap_exe):
-            os.unlink(bootstrap_exe)
-        os.rename('ninja.exe', bootstrap_exe)
-        subprocess.check_call('ninja.bootstrap.exe%s' % verbose, shell=True)
+        final_exe = 'ninja.exe'
     else:
-        subprocess.check_call('./ninja%s' % verbose, shell=True)
+        bootstrap_exe = './ninja_bootstrap'
+        final_exe = './ninja'
+    if os.path.exists(bootstrap_exe):
+        os.unlink(bootstrap_exe)
+    os.rename(final_exe, bootstrap_exe)
+    subprocess.check_call(bootstrap_exe + verbose, shell=True)
```

Comment:
Yes, it has the benefit that you're manually testing the ninja binary you're working on all the time.